.changeset/modern-hornets-tell.md

```diff
@@ -0,0 +1,8 @@
+---
+"@turnkey/sdk-browser": major
+"@turnkey/sdk-server": major
+"@turnkey/sdk-react": minor
+"@turnkey/http": minor
+---
+
+InitOtpV2 allows alphanumeric boolean and otpLength (6-9) to be passed + associated updates to EWK
```

Comment:
Let's add some notes on the http changes as well. Can be done in the release PR

Reply:
e.g. the addition of `sendFromEmailSenderName` to some activities. This is what I have in mind: https://github.com/tkhq/sdk/blob/main/packages/http/CHANGELOG.md#minor-changes